dask/delayed.py

```diff
@@ -520,6 +520,10 @@ def __call__(self, *args, **kwargs):
     Delayed._bind_operator(op)
 
 
+if PY3:
+    Delayed._bind_operator(operator.matmul)
```

Comment:
Guess we forgot about Python 3.4 (myself included). Put up a tweak in PR ( https://github.com/dask/dask/pull/3791 ), which should allow this to work on Python 3.4 as well.